src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -3565,9 +3525,9 @@ ExplicitInterfaceSpecifierSyntax tryParseExplicitInterfaceSpecifier()
                     // now, scan past the next name.  if it's followed by a dot then
                     // it's part of the explicit name we're building up.  Otherwise,
                     // it should be an operator token
-                    var point = GetResetPoint();
+
                     bool isPartOfInterfaceName;
-                    try
+                    using (var _ = GetDisposableResetPoint(resetOnDispose: true))
```

Comment:
`using (GetDisposableResetPoint(resetOnDispose: true))` should work (simpler)

Reply:
simplified.